pyproject.toml

```diff
@@ -1,6 +1,55 @@
 [build-system]
-requires = ["setuptools>=41.0", "wheel", "setuptools_scm[toml]>=3.4"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling>=1.8.0", "hatch-vcs", "hatch-fancy-pypi-readme"]
+build-backend = "hatchling.build"
+
+[project]
+name = "bo4e-python"
+description = "Python Library that implements the BO4E Standard."
+license = { text = "MIT" }
+requires-python = ">=3.10"
+authors = [{ name = "Kevin Krechan", email = "kevin.krechan@hochfrequenz.de" }]
+keywords = ["bo4e", "bdew", "edi@energy"]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+]
+dependencies = [
```

Comment:
magst du hier den hinweis einbauen, dass leider immer noch requirements.in und diese liste händisch in-sync gehalten werden müssen?

Reply:
done in 8ddbcda9ac1e6ed134bc22f51030be4d8357e5a5